jumpy/pom.xml

```diff
@@ -4,12 +4,12 @@
     <groupId>org.deeplearning4j</groupId>
     <artifactId>dl4j</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.0-beta</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Get rid of the url and change the artifactId to jumpy ?

Reply:
Sorry, which URL?

Reply:
Ah, I think I found URL :)